jlm/llvm/ir/operators/MemCpy.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Nico Reißmann <nico.reissmann@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include <jlm/llvm/ir/operators/MemCpy.hpp>
+
+namespace jlm::llvm
+{
+
+MemCpyOperation::~MemCpyOperation() = default;
+
+bool
+MemCpyOperation::operator==(const operation & other) const noexcept
+{
+  // Avoid common node elimination for memcpy operator
```

Comment:
Why is this important?

Reply:
It was (not) there already. I implemented now the equality operator.